src/Adapter/MSTest.TestAdapter/Discovery/AssemblyEnumerator.cs

```diff
@@ -160,7 +160,7 @@ internal static string GetLoadExceptionDetails(ReflectionTypeLoadException ex)
         var map = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase); // Exception -> null.
         var errorDetails = new StringBuilder();
 
-        if (ex.LoaderExceptions != null)
+        if (ex.LoaderExceptions?.Length > 0)
```

Comment:
Looks like a change of behavior on .NET where we now have empty array instead of null.